src/components/reports/metrics-report.js

```
@@ -13,14 +13,12 @@
-  DateTimePicker,
-  Dropdown,
-  EventInput,
-  Panel,
-  Table
-} from "openstack-uicore-foundation/lib/components";
+import CompanyInput from "openstack-uicore-foundation/lib/components/inputs/company-input"
+import DateTimePicker from "openstack-uicore-foundation/lib/components/inputs/datetimepicker"
+import Dropdown from "openstack-uicore-foundation/lib/components/inputs/dropdown"
+import EventInput from "openstack-uicore-foundation/lib/components/inputs/event-input"
+import Panel from "openstack-uicore-foundation/lib/components/sections/panel"
+import Table from "openstack-uicore-foundation/lib/components/table";
```

ESLint

[error] Expected 1 empty line after import statement not followed by another import.

(import/newline-after-import)